sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -399,7 +399,7 @@ def from_connection_string(
             raise ValueError("Subscription name is missing for the topic. Please specify subscription_name.")
         return cls(**constructor_args)
 
-    def receive_messages(self, max_message_count=None, max_wait_time=None):
+    def receive_messages(self, max_message_count=1, max_wait_time=None):
```

Comment:
`async receive_messages` should apply the change as well which is here:
https://github.com/Azure/azure-sdk-for-python/blob/7a730caaeb6ca118a95b86df08448b2aac9713ae/sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py#L396